cookiecutter.json

```diff
@@ -10,7 +10,10 @@
   "git_remote_url": "",
   "version": "0.0.1",
   "python_version": "3.11",
-  "is_ci": ["no", "yes"],
+  "is_ci": [
```

Comment:
Just to be sure: The order does not matter here at all, right? I'll move it to the very end here then and make a couple of other changes, too.